ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/snap/SnapProtocolManager.java

```diff
@@ -123,18 +126,23 @@ public void processMessage(final Capability cap, final Message message) {
               .dispatch(new EthMessage(ethPeer, requestIdAndEthMessage.getValue()))
               .map(responseData -> responseData.wrapMessageData(requestIdAndEthMessage.getKey()));
     } catch (final RLPException e) {
-      LOG.debug(
-          "Received malformed message {} , disconnecting: {}", messageData.getData(), ethPeer, e);
+      LOG.atDebug()
```

Comment:
this is a perfect example, there is no overhead in the old statement, and it is more compact and easier to read